src/cljs/netrunner/gameboard.cljs

```diff
@@ -327,18 +326,15 @@
             height (.-scrollHeight div)]
         (when (or (zero? scrolltop)
                   (< (- height scrolltop (.height (js/$ ".gameboard .log"))) 500))
-          (aset div "scrollTop" height))))
-
-
-    om/IDidMount
+          (aset div "scrollTop" height)))) om/IDidMount
```

Comment:
That's still valid code, since this is just a list of arguments, but I'll fix it.